### DISCUSSION TITLE
NG8002: Can't bind to 'selectionKeys' since it isn't a known property of 'p-treeTable'.

### DISCUSSION THREAD

**Discussion by mahraz-hasnat** [primefaces/community#2675] [PrimeNG]
Getting this error _NG8002: Can't bind to 'selectionKeys' since it isn't a known property of 'p-treeTable'._ though I already have imported `TreeTableModule`. I was trying to implement this https://primeng.org/treetable#checkbox in my project. I'm using angular 17

**Comment by qwadrox** [comment]
Hi @mahraz-hasnat! Could create a reproduction sample with stackblitz?
